README.md

```diff
@@ -67,7 +65,7 @@ statful.counter('transactions', 1);
 
 > **IMPORTANT:** This configuration uses the default **host** and **port**. You can find these default values in the below table. 
```

Comment:
@SujathaRamanMindera probably add something on this paragraph to mention that the transport attribute influences the default values applied.